example_configs/kafka-connect.yml

```diff
@@ -0,0 +1,100 @@
+lowercaseOutputName: true
+rules:
+  #kafka.connect:type=app-info,client-id="{clientid}"
+  #kafka.consumer:type=app-info,client-id="{clientid}"
+  #kafka.producer:type=app-info,client-id="{clientid}"
+  - pattern: 'kafka.(.+)<type=app-info, client-id=(.+)><>([a-z-]+): (.+)'
+    name: kafka_$1_appinfo
+    value: 1
+    labels:
+      client: "$2"
+      $3: "$4"
+    help: "Kafka $1 JMX metric info $2 $3"
+    type: UNTYPED
+
+  #kafka.connect:type=connect-node-metrics,client-id="{clientid}",node-id="{nodeid}"
+  #kafka.consumer:type=consumer-node-metrics,client-id=consumer-1,node-id="{nodeid}"
+  - pattern: kafka.(.+)<type=(.+)-metrics, client-id=(.+), node-id=(.+)><>([a-z-]+)
+    name: kafka_$1_$2_$5
+    labels:
+      clientId: "$3"
+      nodeId: "$4"
```

Comment:
If this metrics belonging to something other than the process itself?

Reply:
The Kafka connect has consumers and producers that exporter metrics JMX. This regex takes metrics of consumer and connects that have similarly MBeans.